### PR TITLE
Adds detail on contribution model

### PR DESCRIPTION
**Pull request by js-kyle**
Introduces details on the new contribution model. Also includes a new presentation of the members in the Readme file. Tasked from issue #319 

image

image

### CODE REVIEW

File: .operations/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Contribution model
+
+### Steering committee 🏆
+
+The steering committee work together to provide guidance and future direction to the project. Each committee member has a particular experitse which they share their knowledge on, and work to lead further improvements to the project in that are. In addition, each member of the committee leads a project which are tracked under Github projects.
```
Comment (BrunoScheufler) [inline .operations/CONTRIBUTING.md]: I'm not sure about the `work together` of the committee language-wise, is the word used in the sense of plural or wouldn't you rather use 'the steering committee works together'? I'd be more familiar with something like the latter or "Members of the steering committee work together..."

Comment (BrunoScheufler) [inline .operations/CONTRIBUTING.md]:
```suggestion
The steering committee work together to provide guidance and future direction to the project. Each committee member has a particular expertise which they share their knowledge on, and work to lead further improvements to the project in that are. In addition, each member of the committee leads a project which are tracked under Github projects.
```

File: .operations/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+### Collaborators 👍
+
+Collaborators are members who are contributing to the repository on a reguar basis, through suggesting new best practices, triaging issues, reviewing pull requests and more. They help the steering committee ensure that the content provided is of a high standard, up to date with the industry, and available in many languages. Members who are displaying the above activity, and quality of contribution will be invited to become a collaborator.
```
Comment (BrunoScheufler) [inline .operations/CONTRIBUTING.md]:
```suggestion
Collaborators are members who are contributing to the repository on a reguar basis, through suggesting new best practices, triaging issues, reviewing pull requests and more. They help the steering committee to ensure that the content provided is of a high standard, up-to-date with the industry, and available in many languages. Members who are displaying the above activity, and quality of contribution will be invited to become a collaborator.
```

File: .operations/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+On the occasion where a collaborator becomes inactive for some time, they will be asked if they would still lik, however will always be acknowledged as a past collaborator.
```
Comment (BrunoScheufler) [inline .operations/CONTRIBUTING.md]:
```suggestion
On the occasion where a collaborator becomes inactive for some time, they will be asked if they would still like to contribute as a collaborator or leave their role, being acknowledged as a past collaborator.
```

Comment (BrunoScheufler) [reply .operations/CONTRIBUTING.md]: The change suggestion above is just that, a suggestion. I'm not completely sure how to rephrase this sentence, maybe we could find a common solution 👍

File: README.md [hunk 4]
```diff
@@ -22,6 +22,8 @@ Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chines
 
 <br/>
 
+###### Built and maintained by our [Steering Committee ](#steering-committee) and [Collaborators](#collaborators)
```
Comment (BrunoScheufler) [inline README.md]: Is there any reason for the trailing space in the "Steering Committee" link? 😄

File: README.md [hunk 5]
```diff
@@ -989,42 +991,71 @@ All translations are contributed by the community. We will be happy to get any h
...
 - ![TR](/assets/flags/TR.png) Turkish ([Discussion](https://github.com/i0natan/nodebestpractices/issues/139))
 
-<br/><br/><br/>
+<br/><br/>
 
-# Core Contributors
+## Steering Committee
 
-## `Yoni Goldberg`
+Meet the steering committee members - the people who work together to provide guidance and future direction to the project. In addition, each member of the committee leads a project which are tracked under [Github projects](https://github.com/i0natan/nodebestpractices/projects).
```
Comment (BrunoScheufler) [inline README.md]:
```suggestion
Meet the steering committee members - the people who work together to provide guidance and future direction to the project. In addition, each member of the committee leads a project tracked under [GitHub projects](https://github.com/i0natan/nodebestpractices/projects).
```

Same as the suggestion above 😄

**Comment by js-kyle** [comment]
Thanks @BrunoScheufler, will take a look at your comments. Looks like I need to install a spellcheck on vscode :)

File: .operations/CONTRIBUTING.md (hunk 3 above)
Comment (js-kyle) [reply .operations/CONTRIBUTING.md]: Whoops, looks like I missed finishing that sentence

**Comment by js-kyle** [comment]
@BrunoScheufler I agreed with your comments - pushed the changes.

Will let the others add their comments!

**Comment by i0natan** [comment]
@js-kyle I'm on a trip until Friday, this PR needs full attention, I will have a deep dive right upon my return 

(A message to bot below)

/remind me in to comment in 2 days

File: .operations/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+### Steering committee 🏆
+
+Members of the steering committee work together to provide guidance and future direction to the project. Each committee member has a particular expertise which they share their knowledge on, and work to lead further improvements to the project in that area. In addition, each member of the committee leads a project tracked under our Github projects.
+
+### Collaborators 👍
+
+Collaborators are members who are contributing to the repository on a regular basis, through suggesting new best practices, triaging issues, reviewing pull requests and more. They help the steering committee to ensure that the content provided is of a high standard, up-to-date with the industry, and available in many languages. Members who are displaying the above activity, and quality of contribution will be invited to become a collaborator.
+
+On the occasion where a collaborator becomes inactive for some time, they will be asked if they would still like to contribute as a collaborator or leave their role, being acknowledged as a past collaborator.
```
Comment (sagirk) [inline .operations/CONTRIBUTING.md]: nit: Rephrase to better reflect the process. Add a missing clause for past collaborators to be able to request to be restored to active status.

```suggestion
The steering committee periodically reviews the collaborator list to identify inactive collaborators. Inactive collaborators can choose to either continue in or step down from their role, in which case they are acknowledged as a past collaborator. They may later request that the steering committee restore them to active status.
```

**Comment by sagirk** [comment]
Great work, @js-kyle! I really love the clean and simple design. 💯

**Comment by i0natan** [comment]
@js-kyle Great work. Added a few comments/questions.

Two other thoughts that just pop to mind:
1. Try to include some exciting and emotional words that encourage people to join. For example: "Join our team to influence thousands in writing better Node applications"

2.  Consider placing icons to the left of each section to keep consistent formatting which resembles bullets

**Comment by i0natan** [comment]
@js-kyle Looks great. One major issue - It doesn't look that good to have a huge steering comittee and one collaborator... should have been the opposite...

I would at least move Ido to collaborators and add write in the description "Found of this repository"

Let's get soon new collaborators....

**Comment by i0natan** [comment]
@js-kyle I guess you don't need anything else from us, right?

**Comment by js-kyle** [comment]
@i0natan no - I will make that change today!

**Comment by js-kyle** [comment]
@i0natan have updated

**Comment by js-kyle** [comment]
@i0natan I've just fixed a merge conflict, keen to merge this if you or someone else can re-approve

**Comment by i0natan** [comment]
@js-kyle Done!

**Pull request merged**